src/main/menu.ts

```diff
@@ -176,6 +176,30 @@ export default class MenuBuilder {
         },
         enabled: false, // by default, gets updated when a project is entered
       },
+      {
+        id: 'playPause',
+        label: 'Play Or Pause Video',
+        accelerator: 'Space',
+        click: () => {
+          this.mainWindow.webContents.send('toggle-play-pause');
+        },
+      },
+      {
+        id: 'skipForward',
+        label: 'Skip forwards n seconds',
+        accelerator: 'Right',
+        click: () => {
+          this.mainWindow.webContents.send('initiate-skip-forward');
+        },
+      },
+      {
+        id: 'skipBackward',
+        label: 'Skip backwards n seconds',
```

Comment:
Same deal here

Reply:
Yeah I remember you telling me to do this earlier, I just forgot to do it, cheers for the reminder